assets/js/modules/adsense/components/common/AdBlockingRecoveryToggle.js

```diff
@@ -160,12 +184,18 @@ export default function AdBlockingRecoveryToggle() {
 					</div>
 				) }
 			</div>
-			{ adBlockingRecoveryToggle === false && (
+			{ ! existingAdBlockingRecoveryTag &&
```

Comment:
Instead of showing both notices here, it makes sense to me to only show the existing tag notice here, thoughts?

Reply:
Sounds reasonable